sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -1,3 +1,29 @@
+public partial class FullBackupOperation : Azure.Operation<Azure.Security.KeyVault.Administration.Models.FullBackupDetails>
```

Comment:
Do we need the `Full` prefix?

Reply:
Arguably not. The only reason I can think of not to is that this is a differentiator between individual [Key level backup](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs#L818) implemented in `KeyClient`. But the context is probably enough of a hint.

Reply:
Should be fine given that `Backup` and `Restore` for `KeyClient` return or take, respectively, a `byte[]` so there's no name conflict.